pypeit/scripts/identify.py

```diff
@@ -94,12 +94,12 @@ def main(args):
 
         # Load the MasterFrame (if it exists and is desired).  Bad-pixel mask
         # set to any flagged pixel in MasterArc.
-        wavecal = BuildWaveCalib(msarc, slits, spec, par, binspectral=slits.binspec, det=args.det,
+        wavecal = BuildWaveCalib(msarc, slits, spec, par, lamplist, binspectral=slits.binspec, det=args.det,
```

Comment:
Can we call this lamps to be consistent with the naming of other variables in other routines?